kyuubi-server/src/main/scala/org/apache/kyuubi/operation/BatchJobSubmission.scala

```diff
@@ -395,11 +395,7 @@ class BatchJobSubmission(
 
   private def cleanupUploadedResourceIfNeeded(): Unit = {
     if (session.isResourceUploaded) {
-      try {
-        Files.deleteIfExists(Paths.get(resource))
-      } catch {
-        case e: Throwable => error(s"Error deleting the uploaded resource: $resource", e)
-      }
+      FileUtils.deleteQuietly(session.resourceUploadFolderPath.toFile)
```

Comment:
You can use `org.apache.kyuubi.Utils.deleteDirectoryRecursively` and remove the common-io dependency.

Reply:
The only concern is that, deleteDirectoryRecursively will break on exception.

Reply:
OK, changed to use `org.apache.kyuubi.Utils.deleteDirectoryRecursively`. And it's made sure it won't break on exceptions and continue to delete the whole folder.